src/main/scala/scopt/options.scala

```diff
@@ -156,13 +156,12 @@ private[scopt] case object Check extends OptionDefKind
  * }}}
  */
 abstract case class OptionParser[C](programName: String) {
-  import OptionDef._
-
   protected val options = new ListBuffer[OptionDef[_, C]]
   protected val helpOptions = new ListBuffer[OptionDef[_, C]]
 
   def errorOnUnknownArgument: Boolean = true
   def showUsageOnError: Boolean = helpOptions.isEmpty
+  def terminationHandler(): Unit = sys.exit
```

Comment:
Maybe it'll be more consistent with other methods if it were a verb phrase, like `terminate` or `exitProgram`.